### DISCUSSION TITLE
Your account has been deactivated due to inactivity.

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/selling-partner-api-models#3428] [General]
I'm using a seller central account only for SP API Access. But amazon deactivates my account due to inactivity. Do I need to sell on amazon or is there any option for developers to avoid deactiation? 

Can anyone help with this?

**Comment by sungolivia** [comment]
I would highly suggest working with our support team directly on this!
https://developer.amazonservices.com/support
